### DISCUSSION TITLE
OPT-071, OPT-072: execution requirements from customer-service list

### DISCUSSION THREAD

**Discussion by odscjen** [OP-TED/eForms-SDK#243] [Q&A] [answered]
OPT-071 and OPT-072 are at XPATH `/*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:TenderingTerms/cac:ContractExecutionRequirement[cbc:ExecutionRequirementCode/@listName='customer-service']` `/cbc:ExecutionRequirementCode` and `/cbc:Description` respectively. However they do not appear in the SDK guidance or any of the example XML's that I checked. Could you provide some guidance as to how these fields are to be used, e.g. can they be used multiple times within the same `ProcurementProjectLot`?

**Comment by rouschr** [comment, marked as answer]
The examples cannot cover every possible case but concerning the **repeatability** of the SDK fields and nodes you have to look into the SDK fields/fields.json file, look for "repeatable".
Either search "OPT-071" or for the xpath:
```
{
    "id" : "OPT-071-Lot",
    "parentNodeId" : "ND-QualityTarget",
    "name" : "Quality Target Code",
    "btId" : "OPT-071",
    "xpathAbsolute" : "/*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:TenderingTerms/cac:ContractExecutionRequirement[cbc:ExecutionRequirementCode/@listName='customer-service']/cbc:ExecutionRequirementCode",
    "xpathRelative" : "cbc:ExecutionRequirementCode",
    "type" : "code",
    "repeatable" : {
      "value" : false,  <----------------
      "severity" : "ERROR"
    }

```
If the field itself is not repeatable look into the "parentNodeId":
```
{
    "id" : "ND-QualityTarget",
    "parentId" : "ND-LotTenderingTerms",
    "xpathAbsolute" : "/*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:TenderingTerms/cac:ContractExecutionRequirement[cbc:ExecutionRequirementCode/@listName='customer-service']",
    "xpathRelative" : "cac:ContractExecutionRequirement[cbc:ExecutionRequirementCode/@listName='customer-service']",
    "repeatable" : true  <------------------
  }
```
The parent node of the field is repeatable (there could be many instances of this), as an analogy you can think about files and folders:
The parent node would be a folder and the fields would be files.
